.travis.yml

```diff
@@ -37,7 +37,7 @@ install:
  # Install Python requirements
  - pip install --upgrade pip setuptools twine
  - pip install --requirement requirements/CI.txt
- - pip install .
+ - pip install --editable .
```

Comment:
i'm still not sure on this editable thing. What is the convention of doing this? Shipping tests might not be that bad? Are the tests changing locations when they are installed without editable?

Reply:
Without `--editable`, `pip` seems to copy the files (including the tests) into `site-packages`. When pytest tries to collect the tests, it will see two different versions of test files: ones in the git repo and ones under `site-packages`. `--editable` solves this issue and I think maybe that should be what we recommend to pymtl3 developers?

Reply:
* `--editable` solves this issue by _not_ copying the files and leaving all source code in the git repo.

Reply:
site-packages is in the virtual environment, right? if you do pytest .. it shouldn't collect tests out of virtual environment

Reply:
It does. [This build](https://travis-ci.com/github/cornell-brg/pymtl3/jobs/311704515) failed because of that and you can use a debug build to see exactly what happened. I can't remember the exact error message but it said two different modules for the test same were found (one under `site-packages` and one in the repo).

Reply:
Why didn't we see this failure before?